minject/inject.py

```diff
@@ -70,6 +84,19 @@ def wrap(cls: Type[T]) -> Type[T]:
     return wrap
 
 
+def async_context(cls: Type[T_async_context]) -> Type[T_async_context]:
+    """
+    Decorator to declare that a class is as an async context manager
+    that can be initialized by the registry through aget(). This
+    is to distinguish the class from an async context manager that
+    should not be initialized by the registry (an example of
+    this being asyncio.Lock()).
```

Comment:
(nit) Drop the parens after `asyncio.Lock`. (`asyncio.Lock` is the type; `asyncio.Lock()` is its constructor function.)